src/router/routes/exportRoute.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-underscore-dangle */
+import express, { Router } from 'express';
+import { ExportType, InitiateExportRequest, Persistence } from 'fhir-works-on-aws-interface';
+import { BadRequestError } from 'fhir-works-on-aws-interface/lib/errors/BadRequestsError';
+import isString from 'lodash/isString';
+import RouteHelper from './routeHelper';
+import ExportHandler from '../handlers/exportHandler';
+
+export default class ExportRoute {
+    readonly router: Router;
+
+    private exportHandler: any;
+
+    private serverUrl: string;
+
+    constructor(serverUrl: string, persistence: Persistence) {
+        this.router = express.Router();
+        this.serverUrl = serverUrl;
+        // @ts-ignore
+        this.exportHandler = new ExportHandler(persistence);
+        this.init();
+    }
+
+    async initiateExportRequests(req: express.Request, res: express.Response, requestGranularity: ExportType) {
+        const requestQueryParams: any = {};
+        // eslint-disable-next-line no-unused-expressions
+        isString(req.query._outputFormat) ? (requestQueryParams._outputFormat = req.query._outputFormat) : '';
+        // eslint-disable-next-line no-unused-expressions
+        Number(req.query._since) ? (requestQueryParams._since = Number(req.query._since)) : '';
+        // eslint-disable-next-line no-unused-expressions
+        isString(req.query._type) ? (requestQueryParams._type = req.query._type) : '';
```

Comment:
It's weird to do assignments like this on a ternary. The usual is:
```js
requestQueryParams._outputFormat = isString(req.query._outputFormat) ? req.query._outputFormat : "";
```

I don't think that there's a good reason to use `// eslint-disable-next-line` here

Reply:
fixed